tests/MongoDB.Driver.Core.Tests/Core/Helpers/MockConnection.cs

```diff
@@ -88,10 +142,18 @@ public List<RequestMessage> GetSentMessages()
 
         public void Open(CancellationToken cancellationToken)
         {
+            _openedAtUtc = DateTime.UtcNow;
+            // _lastUsedAtUtc is set in the SendBuffer method in the BinaryConnection
+            // which is one from methods called inside Open
+            _lastUsedAtUtc = DateTime.UtcNow;
         }
 
         public Task OpenAsync(CancellationToken cancellationToken)
         {
+            _openedAtUtc = DateTime.UtcNow;
+            // _lastUsedAtUtc is set in the SendBufferAsync method in the BinaryConnection
+            // which is one from methods called inside OpenAsync
```

Comment:
Did you mean "which is one of the methods called from inside OpenAsync"?

Reply:
yep, I will fix it